### ISSUE TITLE
TypeError: BinaryReader.seek() takes 2 positional arguments but 3 were given

### ISSUE THREAD

**Issue by SWW13** [Vector35/dwarf_import#18]
```
Traceback (most recent call last):
  File "/opt/binaryninja/plugins/../python/binaryninja/plugin.py", line 229, in _default_is_valid
    return is_valid(view_obj)
  File "/home/user/.binaryninja/repositories/official/plugins/Vector35_dwarf_import/__init__.py", line 86, in is_valid
    return raw and elf and ELFFile(bn.binaryview.BinaryReader(bv.file.raw)).has_dwarf_info()
  File "/home/user/.binaryninja/python310/site-packages/elftools/elf/elffile.py", line 81, in __init__
    self.stream.seek(0, io.SEEK_END)
TypeError: BinaryReader.seek() takes 2 positional arguments but 3 were given
```

Binary Ninja Version: 3.1.3469 Personal, 93650e26
Platform: Arch Linux

**Comment by KyleMiles** [comment]
Could you please tell me what version of pyelftools you have installed?

**Comment by SWW13** [comment]
`extra/python-pyelftools 0.28-1`

**Comment by Mrmaxmeier** [comment]
The relevant pyelftools code was added in [this commit](https://github.com/eliben/pyelftools/commit/45087daac5d4fe661beeb3c8884c50343e8b29a8) and [binaryview.BinaryReader](https://github.com/Vector35/binaryninja-api/blob/dev/python/binaryview.py#L8012) is missing both support for `seek`'s whence argument as well as `tell`. There's a few other functions in Python's [IOBase](https://docs.python.org/3/library/io.html#io.IOBase) that are not implemented by BinaryReader, but not implementing those is probably fine ^^

**Comment by SWW13** [comment]
Turns out BinaryNinja installs it's own Python packages with out any versioning in `.binaryninja/python310/site-packages/` and was using `0.29`.
Removing the package allows the usage of the system package, then I run into #14.

Is this "official" plugin even maintained?

**Comment by KyleMiles** [comment]
This should be resolved in v1.1.2, which you can now get from the plugin manager.
